Layout test results pages should support dark mode

smfr opened the pull request:
#### ab63faf432abcb7cfc469fbb1a0d2a89c94cfa20
<pre>
Layout test results pages should support dark mode
<a href="https://bugs.webkit.org/show_bug.cgi?id=243613">https://bugs.webkit.org/show_bug.cgi?id=243613</a>

Reviewed by Aditya Keerthi.

Add support for light and dark color schemes in the various HTML pages
that are include with layout test results.

* LayoutTests/fast/harness/image-diff-template.html:
* LayoutTests/fast/harness/results.html:
* LayoutTests/fast/harness/test-duration-treemap.html:

Canonical link: <a href="https://commits.webkit.org/253173@main">https://commits.webkit.org/253173@main</a>
</pre>

pxlcoder commented on LayoutTests/fast/harness/test-duration-treemap.html:
```diff
@@ -27,6 +27,54 @@
+            --drop-button-border-color: rgba(128, 128, 128, 0.3);
+            --drop-button-over-border-color: rgba(128, 128, 128, 0.1);
+        }
+        a:link { color: #9e9eff; }
+        a:visited { color: #d0adf0; }
+        a:active { color: red; }
```
Ditto (results.html:20).

pxlcoder commented on LayoutTests/fast/harness/results.html:
```diff
@@ -1,5 +1,27 @@
+        --panel-background-color: rgba(50, 50, 50, 0.9);
+        --clickable-link-color: #9e9eff;
+    }
+    a:link { color: #9e9eff; }
+    a:visited { color: #d0adf0; }
+    a:active { color: red; }
```
I assume these are due to https://bugs.webkit.org/show_bug.cgi?id=209851? (which I really need to look into...)

If so, can we leave a comment with that bug number, so that we can try to remove this CSS in the future?

smfr replied:
Yes, they are.

smfr committed:
Layout test results pages should support dark mode
https://bugs.webkit.org/show_bug.cgi?id=243613

Reviewed by Aditya Keerthi.

Add support for light and dark color schemes in the various HTML pages
that are include with layout test results.

* LayoutTests/fast/harness/image-diff-template.html:
* LayoutTests/fast/harness/results.html:
* LayoutTests/fast/harness/test-duration-treemap.html:

Canonical link: https://commits.webkit.org/253173@main
